plugins/upload-assets-to-s3/src/tasks/upload-assets-to-s3.ts

```diff
@@ -84,8 +84,8 @@ export default class UploadAssetsToS3 extends Task<typeof UploadAssetsToS3Schema
     const files = glob.sync(globFile, { cwd: options.directory })
 
     const s3 = new aws.S3({
-      accessKeyId: options.accessKeyId,
-      secretAccessKey: options.secretAccessKey
+      accessKeyId: process.env[options.accessKeyId],
+      secretAccessKey: process.env[options.secretAccessKey]
```

Comment:
**issue**: these option names are confusing now that they're meant to represent the name of the environment variables that store the access key, rather than the access key itself.

**suggestion**: I think we should deprecate these options (as opposed to deleting them) and replace them with options like `accessKeyIdEnvVar` and `secretAccessKeyEnvVar`. the option values can fall back to the values set in `accessKeyId` and `secretAccessKey` if they aren't set themselves to maintain backwards compatibility